code/modules/client/preference/link_processing.dm

```diff
@@ -1086,6 +1086,10 @@
 					if(parent && parent.mob && parent.mob.hud_used)
 						parent.mob.hud_used.update_parallax_pref()
 
+				if("parallax_darkness")
+					toggles2 ^= PREFTOGGLE_2_PARALLAX_IN_DARKNESS
+					parent?.mob?.hud_used?.update_parallax_pref()
```

Comment:
If a holder somehow has no client, it should probably runtime while running this.
```suggestion
					parent.mob?.hud_used?.update_parallax_pref()
```